_config.yml

```diff
@@ -139,6 +139,12 @@ pwa:
   enabled: true # The option for PWA feature (installable)
   cache:
     enabled: true # The option for PWA offline cache
+    # URLs defined here will be excluded from the PWA cache.
+    # Usually it's value is the part of URL of another website.
+    # If URL includes the value defined, then it will be excluded.
+    deny_urls:
+      - "goatcounter.com/counter"
+      # - "example.com" # URLs match `*example.com*` will not be cached by the PWA
```

Comment:
Actually, there is no need to add a new variable in the `_config.yml`. We can automatically generate `denyUrls` in the `assets/js/data/swconf.js` file.

```suggestion
```

Reply:
Yes, this was optional. Removed config changes.